src/applications/mhv-secure-messaging/components/ComposeForm/ReplyDraftItem.jsx

```diff
@@ -157,7 +157,7 @@ const ReplyDraftItem = props => {
         messageValid = false;
       }
       setMessageInvalid(!messageValid);
-      return messageValid;
+      return { messageValid };
```

Comment:
@vsaleem trying to understand why we return `messageValid` as object while we assign a boolean value to it above? I wonder if this is introducing a defect I am seeing where this does not prevent from firing an API call to send or save during validation?

Reply:
Thanks @oleksii-morgun, this is a great catch! I chose to return `messageValid` as an object from `checkMessageValidity()` so that we can use this value elsewhere in other places. We're checking for `messageValid` when saving messages and I forgot to add this in when sending a reply message as well... This change was really cosmetic, I liked reading the values this way instead of checking for if `checkMessageValidity()` was true or false🙈🙈